examples/mugen/retrieval/model.py

```diff
@@ -4,30 +4,37 @@
 # This source code is licensed under the BSD-style license found in the
 # LICENSE file in the root directory of this source tree.
 
-from typing import Any
+import warnings
+from typing import Any, Tuple
+
+import torch
 
 from examples.mugen.retrieval.video_clip import videoclip
 from pytorch_lightning import LightningModule
+from torchmetrics import Recall
+
 from torchmultimodal.modules.losses.contrastive_loss_with_temperature import (
     ContrastiveLossWithTemperature,
```

Comment:
it's not used anymore?

Reply:
it's not, since i replaced measuring loss during evaluation to measuring recall (i kept the import and args because it will be used when i add training code). i think loss isn't usually reported during evaluation